packages/api/test/functional/technology-order.spec.js

```diff
@@ -0,0 +1,182 @@
+	phone_number: '(99)23456789',
+	lattes_id: '1234567890',
+	address: 'Testing address, 99',
+	address2: 'Complement 99',
+	district: '99',
+	city: 'Test City',
+	state: 'TT',
+	country: 'Fictional Country',
+};
+
+test('PUT /orders/:id/close buyer tryning to close a technology order.', async ({ client }) => {
+	const technologyPurchesed = await Technology.create(technology);
+
+	const buyerUser = await User.create(buyer);
+	const sellerUser = await User.create(seller);
+
+	await technologyPurchesed.users().attach(sellerUser.id);
+
+	const technologyOrder = await TechnologyOrder.create(order);
+	await Promise.all([
+		technologyOrder.technology().associate(technologyPurchesed),
+		technologyOrder.user().associate(buyerUser),
+	]);
+
+	const response = await client
+		.put(`/orders/${technologyOrder.id}/close`)
+		.loginVia(buyerUser, 'jwt')
+		.send({ quantity: 3, unit_value: 100000 })
+		.end();
+
+	response.assertStatus(403);
+	response.assertJSONSubset(
+		errorPayload(errors.UNAUTHORIZED_ACCESS, antl('error.permission.unauthorizedAccess')),
+	);
+});
+
+test('PUT /orders/:id/close user tryning to close a technology order with a non open status.', async ({
+	client,
+}) => {
+	const technologyPurchesed = await Technology.create(technology);
+
+	const buyerUser = await User.create(buyer);
+	const sellerUser = await User.create(seller);
+
+	await technologyPurchesed.users().attach(sellerUser.id);
+
+	const technologyOrder = await TechnologyOrder.create(closedOrder);
+	await Promise.all([
+		technologyOrder.technology().associate(technologyPurchesed),
+		technologyOrder.user().associate(buyerUser),
+	]);
+
+	const response = await client
+		.put(`/orders/${technologyOrder.id}/close`)
+		.loginVia(sellerUser, 'jwt')
+		.send({ quantity: 3, unit_value: 100000 })
+		.end();
+
+	response.assertStatus(400);
+	response.assertJSONSubset(
+		errorPayload(
+			errors.STATUS_NO_ALLOWED_FOR_OPERATION,
+			antl('error.operation.statusNoAllowedForOperation', {
+				op: 'CLOSE ORDER',
+				status: technologyOrder.status,
+			}),
+		),
+	);
+});
+
+test('PUT /orders/:id/close seller closes a technology order.', async ({ client, assert }) => {
+	const technologyPurchesed = await Technology.create(technology);
+
+	const buyerUser = await User.create(buyer);
+	const sellerUser = await User.create(seller);
+
+	await technologyPurchesed.users().attach(sellerUser.id);
+
+	const technologyOrder = await TechnologyOrder.create(order);
+	await Promise.all([
+		technologyOrder.technology().associate(technologyPurchesed),
+		technologyOrder.user().associate(buyerUser),
+	]);
+
+	const response = await client
+		.put(`/orders/${technologyOrder.id}/close`)
+		.loginVia(sellerUser, 'jwt')
+		.send({ quantity: 3, unit_value: 100000 })
+		.end();
+
+	const orderClosed = await TechnologyOrder.find(response.body.id);
+
+	assert.equal(orderClosed.status, orderStatuses.CLOSED);
+
+	response.assertStatus(200);
+	response.assertJSONSubset(orderClosed.toJSON());
```

Comment:
Penso que seria melhor fazermos uma checagem como esta:
```suggestion
	response.assertStatus(200);
	assert.equal(orderClosed.status, orderStatuses.CLOSED);
	assert.equal(response.body.technology_id, technologyPurchased.id);
```